appium/tests/specs/composeEmail/SentEncryptedEmailAfterWrongPassPhrase.spec.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  SplashScreen,
+  CreateKeyScreen,
+  InboxScreen,
+  NewMessageScreen,
+  EmailScreen,
+  MenuBarScreen
+} from '../../screenobjects/all-screens';
+
+import { CommonData } from '../../data';
+import DataHelper from "../../helpers/DataHelper";
+
+describe('COMPOSE EMAIL: ', () => {
+
+  it('user is able to sent encrypted email after reseting pass phrase', async () => {
+
+    const contactEmail = CommonData.secondContact.email;
+    const emailSubject = CommonData.simpleEmail.subject + DataHelper.uniqueValue();
+    const emailText = CommonData.simpleEmail.message;
+    const passPhrase = CommonData.account.passPhrase;
+    const wrongPassPhraseError = CommonData.errors.wrongPassPhrase;
+    const wrongPassPhrase = "wrong";
+    const senderEmail = CommonData.account.email;
+
+
+    await SplashScreen.login();
+    await CreateKeyScreen.setPassPhrase();
+
+    await InboxScreen.clickCreateEmail();
+    await NewMessageScreen.composeEmail(contactEmail, emailSubject, emailText);
+    await NewMessageScreen.checkFilledComposeEmailInfo(contactEmail, emailSubject, emailText);
+    //Set wrong pass phrase and check error
+    await NewMessageScreen.clickSentButton();
+    await EmailScreen.enterPassPhrase(wrongPassPhrase);
+    await EmailScreen.clickOkButton();
+    await NewMessageScreen.checkError(wrongPassPhraseError);
+    await NewMessageScreen.clickOkButtonOnError();
+    //Set correct pass phrase
+    await NewMessageScreen.clickSentButton();
+    await EmailScreen.enterPassPhrase(passPhrase);
+    await EmailScreen.clickOkButton();
+    await EmailScreen.checkSentEmailMessage();
+
+    await MenuBarScreen.clickMenuIcon();
+    await MenuBarScreen.clickSentButton();
+    //Check sent email
+    await InboxScreen.clickOnEmailBySubject(emailSubject);
+    await EmailScreen.checkOpenedEmail(senderEmail, emailSubject, emailText);
+
```

Comment:
In another issue:

It would be good, since we already sent a new message we don't need to also test message actions on it and delete it in the same test. 

- before opening the message, verify it's marked as unread in inbox
- after opening the message, run `checkOpenedEmail` as before
- after that, go back to sent (can use the back icon) and verify it's marked as read
- open the message again and tap "mark as read". Back to sent list - verify it's unread (this checks that we cleverly re-rendered the message after the action without need to refresh)
- refresh the sent message by pulling down on the list, then verify it's still unread (this checks that it was updated as unread on the mail server)
- open the message again and tap "Trash" icon
- go back using back icon, and verify that the message is gone from the list without refreshing
- refresh the list, make sure it's still gone
- using Menu, go to Trash folder, find the message, open it, and tap Trash again (this completely deletes the message)
- using back button, go back to Trash, see that message is not there
- refresh Trash by pulling down, see that message is not there

Then finally, change the test name by adding `+ mark unread, move to trash, delete`